flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -708,6 +708,11 @@ public class TaskManagerOptions {
                                                     "The %s mode tries to spread out the slots evenly across all available %s.",
                                                     code(TaskManagerLoadBalanceMode.SLOTS.name()),
                                                     code("TaskManagers")),
+                                            text(
+                                                    "The %s mode tries to allocate slots on minimum number of available resources.",
+                                                    code(
+                                                            TaskManagerLoadBalanceMode.MIN_RESOURCES
+                                                                    .name())),
```

Comment:
Hi, @vahmed-hamdy Thanks for the contribution !
LGTM overall , only a minor comments.

What about 

```suggestion
                                                    "The %s mode tries to allocate slots on minimum number of available %s.",
                                                    code(
                                                            TaskManagerLoadBalanceMode.MIN_RESOURCES
                                                                    .name()),
                                                    code("TaskManagers")),
```

The description of `TaskManager` sounds intuitive and easy to understand. WDYT ?